lib/engine/game/base.rb

```diff
@@ -654,7 +663,16 @@ def operated_operators
       end
 
       def current_action_id
-        @actions.size + 1
+        @actions.size
+      end
+
+      def last_game_action
```

Comment:
last_game_action_id maybe?

Reply:
I modeled it after `last_processed_action`. Which way do you want to go?

Reply:
_id better describes the object. Changing globally. Also for turn_start_action and last_turn_start_action.